### ISSUE TITLE
Author Submissions Index Page delete/review links should be more descriptive (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#93]
On the author submissions index page, [delete] and [review] links are confusing.

The links should be more descriptive [delete submission], [review program information], [update author information], etc.  Detailed text does not have to be visible to sighted users; hidden text can be used for the screen-reader only if the labeling changes need to be lengthy.
